$NetBSD: patch-Makefile,v 1.1.1.1 2016/06/06 21:31:52 alnsn Exp $

Install config-5.x.lua to LUA_EXAMPLESDIR. Do not undo REPLACE_LUA changes.

--- Makefile.orig	2015-12-29 15:50:56.000000000 +0000
+++ Makefile
@@ -9,7 +9,7 @@ SYSCONFDIR ?= $(PREFIX)/etc/luarocks
 LUA_DIR ?= /usr/local
 LUA_BINDIR ?= $(LUA_DIR)/bin
 
-CONFIG_FILE = $(SYSCONFDIR)/config-$(LUA_VERSION).lua
+CONFIG_FILE = $(PREFIX)/$(LUA_EXAMPLESDIR)/config-$(LUA_VERSION).lua
 
 SAFEPWD=`echo "$$PWD" | sed -e 's/\([][]\)\1/]]..'\''\1\1'\''..[[/g'`
 
@@ -116,7 +116,7 @@ cleanup_bins:
 	for f in $(BIN_FILES) ;\
 	do \
 	   mv src/bin/$$f src/bin/$$f.bak ;\
-	   sed "s,^#!.*lua.*,#!/usr/bin/env lua,;/^package.path/d" < src/bin/$$f.bak > src/bin/$$f ;\
+	   sed "/^package.path/d" < src/bin/$$f.bak > src/bin/$$f ;\
 	   chmod +x src/bin/$$f ;\
 	   rm -f src/bin/$$f.bak ;\
 	done
